tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/CompositeViewsSpec.scala

```diff
@@ -155,14 +153,8 @@ class CompositeViewsSpec extends BaseSpec {
       }
     }
 
-    "wait for data to be indexed after creation" in {
-      Task
```

Comment:
I removed those as they were slowing down the tests, it went from >1 minutes to 20 seconds locally